test/test_packages.py

```diff
@@ -0,0 +1,170 @@
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+"""
+test_packages
+~~~~~~~~~~~~~~~
+This test module tests if R and Python packages installed can be imported.
+It's a basic test aiming to prove that the package is working properly.
+
+The goal is to detect import errors that can be caused by incompatibilities between packages for example:
+
+- #1012: issue importing `sympy`
+- #966: isssue importing `pyarrow`
+
+This module checks dynmamically, through the `CondaPackageHelper`, only the specified packages i.e. packages requested by `conda install` in the `Dockerfiles`.
+This means that it does not check dependencies. This choice is a tradeoff to cover the main requirements while achieving reasonabe test duration.
+However it could be easily changed (or completed) to cover also dependencies `package_helper.installed_packages()` instead of `package_helper.specified_packages()`.
```

Comment:
I agree with the trade-off you've made here, that we should only be testing direct dependencies for the time being. We have a ~50 minute runtime limit on Travis. We'll likely need to finish #693 to reduce the number of images we build, switch CIs, and/or change build strategies if we decide one day to test *everything*.

Reply:
Thanks make sense.